pkg/init/init

```diff
@@ -2,6 +2,14 @@
 
 /bin/mount -t tmpfs tmpfs /mnt
 
+# just copying everything uses a lot of space, so use mv which frees it up
+# copy directory structure under /containers
+find /containers -type d | cpio -p /mnt 2> /dev/null
+# move files under /containers
+find /containers -not -type d -exec mv {} /mnt/{} \;
```

Comment:
The whole point is to remove the file from the old mount point immediately after copying it to the new one, else nothing changes.